tasks/elasticsearch-plugins.yml

```diff
@@ -46,18 +46,20 @@
   environment:
     CONF_DIR: "{{ conf_dir }}"
     ES_INCLUDE: "{{ instance_default_file }}"
+    ES_ENV_FILE: "{{ instance_default_file }}"
```

Comment:
I need to check we need this. Against which version of ES did you test this @orphaner ?

Reply:
5.4.3

Reply:
https://github.com/elastic/elasticsearch/pull/25804/files